lib/api/strapi/generated/sdk.ts

```diff
@@ -136,10 +136,12 @@ export type Collection = {
   children?: Maybe<Array<Maybe<Collection>>>;
   created_at: Scalars['DateTime'];
   description: Scalars['String'];
+  device_title?: Maybe<Scalars['String']>;
```

Comment:
I thought a previous pull was getting rid of this generated file.

Also, I thought we were using graphql-standard camel case field names, rather than snake case.

Reply:
https://github.com/iFixit/react-commerce/pull/126#discussion_r743813345